app/Exceptions/LocationExceptions/UpdateProjectLocationFailed.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Exceptions\LocationExceptions;
+
+use Exception;
+use App\Exceptions\BuildResponse\BuildResponse;
+
+class UserCreatedFailed extends Exception
+{
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Render an exception into an HTTP response.
+     * 
+     * @return \Illuminate\Http\Response
+     */
+    public function render()
+    {
+        $message = 'Project Location was not updated.';
+        $status_code = 500;
```

Comment:
you can also ``` $throw locationWasNotUpdated('project')```
it will be passed in the constructor,
set it as a global variable 
and then you can use it in the message